### PR TITLE
add parameters for bamford buzzard

### PR DESCRIPTION
**Pull request by bobturneruk**
Please merge my species?

### CODE REVIEW

File: params/bamford_buzzard.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.5
+    
+# e.g. "anas_krystallinus"    
+species.name <- "bamford_buzzard"
```
Comment (ns-rse) [inline params/bamford_buzzard.R]: Thanks @bobturneruk for the contribution, we've had a few submissions of Bamford buzzards recently, would you mind changing the `species.name` please?

Comment (g1tKZ) [inline params/bamford_buzzard.R]: Thanks @bobturneruk  for the contribution. Would you mind changing the name?

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks, I've noticed a few buzzards from Bamford, would you mind changing the `species.name` please?

File: params/bamford_buzzard.R (hunk 1 above)
Comment (bobturneruk) [reply params/bamford_buzzard.R]: Good spot, I've renamed the species.

**Review by ns-rse — APPROVED** [review]
They get around those buzzards, thanks for the contribution @bobturneruk  looks like its ready to merge.

@ns-rse

**Pull request merged**